code.cpp

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
The code should be inside a folder with the name of the algorithm.
Please check the CONTRIBUTING GUIDELINES.

Reply:
Yes my bad, edited that!